pydantic/types.py

```diff
@@ -51,52 +51,52 @@
 )
 
 __all__ = [
-    "NoneStr",
```

Comment:
@ samuelcolvin prefers single quotes, so you'll need to undo this and rerun with `-S`, as specified in https://github.com/samuelcolvin/pydantic/blob/master/Makefile#L3

It would be better in fact to simply run `make format`, which will give all the required parameters needed.

Reply:
I think you've reviewed an old commit :thinking:

Reply:
So I did. Carry on!